lib/lndclient/LndClient.ts

```diff
@@ -387,6 +414,34 @@ class LndClient extends SwapClient {
     return this.unaryCall<lndrpc.SendToRouteRequest, lndrpc.SendResponse>('sendToRouteSync', request);
   }
 
+  public genSeed = async (): Promise<lndrpc.GenSeedResponse.AsObject> => {
+    const genSeedResponse = await this.unaryWalletUnlockerCall<lndrpc.GenSeedRequest, lndrpc.GenSeedResponse>(
+      'genSeed', new lndrpc.GenSeedRequest(),
+    );
+    return genSeedResponse.toObject();
+  }
+
+  public initWallet = async (walletPassword: string, seedMnemonic: string[]): Promise<lndrpc.InitWalletResponse.AsObject> => {
+    const request = new lndrpc.InitWalletRequest();
```

Comment:
What do you think about extending `seedutil` to generate the seed for us? We already have `aezeed` package imported and it would be really trivial to extend `seedutil` to support `seedutil create [optinal-pw]`. That way we wouldn't have to rely on lnd to be enabled, connected AND in a specific state.

Reply:
Nevermind, we still need to `init` lnd wallet somehow.

Reply:
This comment is still valid for `swapClientManager.genSeed`, though.

Reply:
You mean using the seedutil to generate a seed rather than returning `undefined`? I guess in that case I'm still thinking there might not be much value since we couldn't use that seed for any lnd instances. Maybe I'll just do that if we're only connected to raiden.

Reply:
Exactly - fallback to `seedutil create`. I'm good with leaving a TODO right now as our current setup assumes lnd is always there.